src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetCultureCommand.cs

```diff
@@ -1,23 +1,94 @@
 using System.Management.Automation;
 
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    /// Returns the thread's current culture.
+    /// Returns:
+    ///     - the thread's current culture
+    ///     - culture by name
+    ///     - list of all supported cultures.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "Culture", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
+    [Cmdlet(VerbsCommon.Get, "Culture", DefaultParameterSetName = DefaultParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
     [OutputType(typeof(System.Globalization.CultureInfo))]
     public sealed class GetCultureCommand : PSCmdlet
     {
+        private const string DefaultParameterSet = "Default";
+        private const string NameParameterSet = "Name";
+        private const string ListParameterSet = "List";
+
+        /// <summary>
+        /// Gets or sets culture names for which CultureInfo values are returned.
+        /// </summary>
+        [Parameter(ParameterSetName = NameParameterSet, Position = 0, ValueFromPipeline = true)]
+        [ValidateNotNull]
+        public string[] Name { get; set; }
+
         /// <summary>
-        /// Output the current Culture info object.
+        /// Gets or sets a switch to return predefined cultures.
+        /// By default we return cultures in current state (with custom changes).
+        /// With the switch on, we return predefined, original cultures.
         /// </summary>
-        protected override void BeginProcessing()
+        [Parameter(ParameterSetName = NameParameterSet)]
+        [ValidateNotNull]
+        public SwitchParameter Predefined { get; set; }
+
+        /// <summary>
+        /// Gets or sets a filter to list subset or all available cultures.
+        /// </summary>
+        [Parameter(ParameterSetName = ListParameterSet)]
+        public CultureTypes List { get; set; } = CultureTypes.AllCultures;
+
+        /// <summary>
+        /// Output:
+        ///     - the thread's current culture
+        ///     - culture by name
+        ///     - list of all supported cultures.
+        /// </summary>
+        protected override void ProcessRecord()
         {
-            WriteObject(Host.CurrentCulture);
+            switch (ParameterSetName)
+            {
+                case DefaultParameterSet:
+                    WriteObject(Host.CurrentCulture);
+
+                    break;
+                case NameParameterSet:
+                    try
+                    {
+                        foreach (var cultureName in Name)
+                        {
+                            CultureInfo ci;
+
+                            if (Predefined)
+                            {
+                                ci = new CultureInfo(cultureName, true);
+                            }
+                            else
+                            {
+                                ci = CultureInfo.GetCultureInfo(cultureName);
```

Comment:
Given my discussions in the CoreFx repo, I still feel strongly that we should validate each `cultureName` against the list of predefined cultures reported by `CultureInfo.GetCultures(CultureTypes.AllCultures)`. 

There is no good reason to follow the - IMO misguided - CoreFx behavior. No one should expect a cmdlet named _Get_-Culture to _create_ cultures on demand, likely unintentionally. It is much more useful to be notified of having accidentally typed the name of a non-predefined culture by way of an error.

We can allow ad-hoc creation on an opt-in basis with another switch, say `-CreateOnDemand`, but I'm not sure that's even necessary.

Reply:
Agree, based on @PowerShell/powershell-committee discussion we should only return cultures from the AllCultures list.  Creation is out of scope of this PR.